docs/imaging_model/README.md

```diff
@@ -0,0 +1,47 @@
+# Imaging and Report Model
+
+## Motivation
+
+Participants often have files in addition to genomic files ranging from
+MRI images, histology images, and reports.
+Some of these file types relate directly to the biospecimen rather than the
+participant, eg: proteomics and histology.
+In order to capture and distribute these files, we need to model them
+appropriately.
+
+## Implementation
+
+Files will need to be distributed through gen3, so the models will need to
+inherit the Indexd model similar to GenomicFiles.
+
+![Imaging Model](ImagingModelTables.png)
+
+### ACLs
+
+These file types are typically not submitted to dbGaP and so can not be
+directly controlled through accession numbers and consents.
+Instead, these files will have ACLs consisting of the study kf_id and the
+most restricted set of consent codes for that participant's biospecimen.
+
+### Migration of current files
+
+Many MRI files have already been loaded as Genomic Files.
+These files will need to be moved to RadiologyFiles.
+Because this was intentionally done as a temporary solution, we will not make
+an attempt to preserve the kf_ids given to the files.
+
+
+## Difficulties
+
+These models and data files are currently only supported by CBTTC with
+only one additonal study in the future.
+Will this model be utilized at some point across more studies, or is it
```

Comment:
I don't think that this matters except for prioritizing development. The design should make sense regardless of how many studies end up contributing this kind of data.

Reply:
I think the main thing for here is that CBTTC is cancer, and so this is cancer-focused with the pathology slides and MRIs. So it should make sense for other (likely cancer) studies that have those modalities of data. Why I think it would be useful to look at TCIA (http://www.cancerimagingarchive.net/) . However, I expect other imaging, such as photographs for phenotyping to require further development in the future.